net/libfab/libfab.c

```diff
@@ -412,7 +416,7 @@ static int libfab_ep_addr_decode_lnet(const char *name, char *node,
 			"portal: %u, tmid: %u", portal, tmid);
 	*/
 
-	if ( portal < 30)
+	if (portal < 30)
```

Comment:
There is still code duplication and proliferation of address formats that could be reduced.

Something like the following can help:

```
/*
 * Take addr_parse_native() and create a new function from it. Use this function
 * in sock.c and libfabric.c to parse ip addresses.
 */

/* New files net/ip.[ch] */
struct m0_net_ip_addr {
	int                     nia_family; /* AF_INET, AF_INET6 */
	int                     nia_stype;  /* SOCK_STREAM, SOCK_DGRAM */
	struct sockaddr_storage nia_saddr;
};

/**
 * Parses IP addresses, for example
 *
 *  "inet:stream:lanl.gov@23",
 *  "inet6:dgram:FE80::0202:B3FF:FE1E:8329@6663" or
 *  "unix:dgram:/tmp/socket".
 *
 * Based on net/sock/sock.c:addr_parse_native().
 */
int m0_net_ip_parse(const char *addr, struct m0_net_ip_addr *addr);

/*
 * Similarly, take sock.c:addr_print() and create a new function, use it in
 * sock.c and libfabric.c to convert ip addresses to strings.
 */
char *m0_net_ip_print(const struct m0_net_ip_addr *nia);

/*
 * Take sock.c:addr_parse_lnet() and create out of it
 */
int m0_net_ip_lnet_parse(const char *addr, struct m0_net_ip_addr *addr);
```
For portals less than 30 apply https://github.com/Seagate/cortx-hare/commit/291353 to cortx-hare.

Use this common-ip and lnet formats for libfabric and sock, please drop "libfab:[...]" format. Later we will introduce a generic function in net.[ch] that will determine transport by prefix.